vyper/utils.py

```diff
@@ -72,15 +72,16 @@ def calc_mem_gas(memsize):
 
 # Number of bytes in memory used for system purposes, not for variables
 class MemoryPositions:
-    RESERVED_MEMORY = 320
     ADDRSIZE = 32
     MAXNUM = 64
     MINNUM = 96
     MAXDECIMAL = 128
     MINDECIMAL = 160
     FREE_VAR_SPACE = 192
-    BLANK_SPACE = 224
-    FREE_LOOP_INDEX = 256
+    FREE_VAR_SPACE2 = 224
+    BLANK_SPACE = 256
+    FREE_LOOP_INDEX = 288
+    RESERVED_MEMORY = 320
```

Comment:
@jacqueswww Is it intended to have the same size of the reserved memory even if we have the additional one (`FREE_VAR_SPACE2`)?

Reply:
@daejunpark there seemed to have been an extra slot btween 288 and 320 that was never used. So I took the liberty of just shifting the addresses up.